app/src/main/java/io/neurolab/main/output/audio/AudioFeedback.java

```diff
@@ -0,0 +1,198 @@
+package io.neurolab.main.output.audio;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Point;
+
+import io.neurolab.main.output.feedback.Feedback;
+import io.neurolab.model.Config;
+import io.neurolab.settings.FeedbackSettings;
+import io.neurolab.tools.ResourceManager;
+
+import java.io.IOException;
+
+import com.jsyn.JSyn;
+import com.jsyn.Synthesizer;
+import com.jsyn.data.FloatSample;
+import com.jsyn.devices.AudioDeviceFactory;
+import com.jsyn.ports.QueueDataCommand;
+import com.jsyn.unitgen.LineOut;
+import com.jsyn.unitgen.VariableRateDataReader;
+import com.jsyn.unitgen.VariableRateMonoReader;
+import com.jsyn.unitgen.VariableRateStereoReader;
+import com.jsyn.util.SampleLoader;
+
+public class AudioFeedback extends Feedback {
+
+    public float masterVolume = .95f;
+    double oldFeedbacks[];
+    double lastFeedback = 0d;
+    long lastTimestamp = 0;
+    int minimumTime = 20;
+    private Context context;
+    private boolean multiModelFeedback = false;
+    private Synthesizer synth;
+    private LineOut lineOut;
+    private FloatSample[] sample;
+    private VariableRateDataReader[] samplePlayer;
+    private int numSamples = 5;
+    private float[] volume = {.5f, .5f, .5f, .5f, .7f};
+    private float[] defaultVolume = {.5f, .5f, .5f, .5f, .7f};
+    private String[] soundMixSamples = {"audio/pad_.wav", "audio/pad1.wav", "audio/lownoise.wav", "audio/pad2.wav", "audio/forest.wav"};
+    private Bitmap feedbackMixImage;
+    private int windowSize = 400;
```

Comment:
I think these parameters are much related to the UI for the Feedback. No need of these in the base class. We can make UI related changes in the Activity and Fragment @suresh-gandhi .